jmix-email/email-flowui/src/main/resources/io/jmix/emailflowui/view/sendingmessage/sending-message-view.xml

```diff
@@ -72,22 +89,23 @@
                                 type="list_itemTracking" icon="lumo:reload"/>
                     </actions>
                     <columns resizable="true">
-                        <column property="address" sortable="false"/>
+                        <column property="address" sortable="false" autoWidth="true"/>
                         <column property="cc"/>
                         <column property="bcc"/>
-                        <column property="from"/>
-                        <column property="subject"/>
-                        <column property="status"/>
-                        <column property="bodyContentType"/>
-                        <column property="createTs"/>
-                        <column property="dateSent"/>
-                        <column property="updateTs"/>
-                        <column property="attachmentsName" sortable="false"/>
-                        <column property="attemptsLimit"/>
-                        <column property="deadline"/>
-                        <column property="attemptsMade"/>
-                        <column property="important"/>
+                        <column property="from" autoWidth="true"/>
+                        <column property="subject" width="300px"/>
+                        <column property="status" autoWidth="true"/>
+                        <column property="bodyContentType" width="180px"/>
```

Comment:
Due to the number of columns we have horizontal scrolling. Does it make sense to truncate the content type value?